src/manager/channels-manager.css

```diff
@@ -100,7 +100,7 @@ main {
     color: var(--accent-light-text);
 }
 
-.selectableItemsList > option:hover:not(:last-child) {
+.selectableItemsList > option:hover {
```

Comment:
Any specific reason to change this? The reason this border isn't shown is the same why the top one is not shown for the first child.

Reply:
The first channel's top border does interfere with the toolbar's bottom border but the last channel's bottom border doesn't intefere with anything. Currently, if yout channel's list doesn't reach the bottom of the page, the last channel visually behaves differently from the others when hovered.